Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+# Math Fundamentals
+
+# What you will learn 
+In this guide, you'll learn what is the fundamental mathematical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
```

Comment:
Does the reader know what Paradiso guides are? And whether it is a higher or lower level guide?

Reply:
Yes they do, it's in the home readme.md 

BTW I-ll do an article about the Purgatorio, at the Purgatorio entry

Reply:
Please do that but back-linking them to the README page will only help and clarify to the readers, it won't take away anything that already is present in the guide.

Reply:
done